src/latexify/latexify_visitor.py

```diff
@@ -246,27 +247,27 @@ def _wrap(child):
             return reprs[type(node.op)]()
         return r"\mathrm{unknown\_binop}(" + _unwrap(lhs) + ", " + _unwrap(rhs) + ")"
 
+    _compare_ops: ClassVar[dict[type[ast.cmpop], str]] = {
+        ast.Eq: "=",
+        ast.Gt: ">",
+        ast.GtE: r"\ge",
+        ast.In: r"\in",
+        ast.Is: r"\equiv",
+        ast.IsNot: r"\not\equiv",
+        ast.Lt: "<",
+        ast.LtE: r"\le",
+        ast.NotEq: r"\ne",
+        ast.NotIn: r"\notin",
+    }
+
     def visit_Compare(self, node, action):  # pylint: disable=invalid-name
         """Visit a compare node."""
-        lstr = self.visit(node.left)
-        rstr = self.visit(node.comparators[0])
-
-        if isinstance(node.ops[0], ast.Eq):
-            return lstr + "=" + rstr
-        if isinstance(node.ops[0], ast.Gt):
-            return lstr + ">" + rstr
-        if isinstance(node.ops[0], ast.Lt):
-            return lstr + "<" + rstr
-        if isinstance(node.ops[0], ast.GtE):
-            return lstr + r"\ge " + rstr
-        if isinstance(node.ops[0], ast.LtE):
-            return lstr + r"\le " + rstr
-        if isinstance(node.ops[0], ast.NotEq):
-            return lstr + r"\ne " + rstr
-        if isinstance(node.ops[0], ast.Is):
-            return lstr + r"\equiv" + rstr
-
-        return r"\mathrm{unknown\_comparator}(" + lstr + ", " + rstr + ")"
+        if len(node.ops) != 1:
+            raise SyntaxError("Multiple compares are not supported.")
```

Comment:
(optional) I think NotImplementedError is slightly better than SyntaxError if we will support the multiple compares in the future.

Reply:
Yeah thanks. I also noticed multiple compares can be easily supported, will fix it in this PR.